src/components/Navbar.astro

```diff
@@ -31,6 +31,7 @@ export const MainNav: RecursiveMenu[] = [
     ]
   },
   { title: 'Knowledge Base', slug: 'https://nfdi4plants.github.io/nfdi4plants.knowledgebase/' },
+  { title: 'Support', slug: '/support' },
```

Comment:
Not a fan of this. As we have all the support stuff on the right side of the navbar:

![image](https://github.com/user-attachments/assets/15fa7c4b-03f1-4170-9d23-21a63c61f195)

Reply:
I would argue socials and support are two very different things. To me it would make a lot of sense also for findability to create an explicit support section. At the very least Matrix should be grouped in with the helpdesk icon, not the social media channels.

Reply:
Done. Please double-check. @Freymaurer